client/src/components/Routes.js

```diff
@@ -13,23 +13,32 @@ import ApplicantProfile from "./ApplicantProfile";
 import CompanyProfile from "./CompanyProfile";
 import OpportunitiesList from "./OpportunitiesList";
 import CompanyRegister from "./CompanyRegister";
+import OpportunityView from "./OpportunityView";
 
 class Routes extends Component {
```

Comment:
The component should be changed to the functional component because you are not using state or React live cycle methods